### PR TITLE
homework-w2 --Shyar

### PR DESCRIPTION
**Pull request by ShyarK**

### CODE REVIEW

File: Week2/queries.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const sqlCountries = `SELECT c.name
+                      FROM country c
+                      JOIN countrylanguage cl 
+                      ON c.code = cl.countryCode
+                      WHERE cl.IsOfficial = 'T'
+                      AND c.region = ?
+                      AND cl.Language = ?`;
+
+const sqlContinentsAndLanguages = `SELECT c.Continent, count(cl.Language) AS Languages
```
Comment (mogaleaf) [inline Week2/queries.js]: What if english is spoken by more than one country in america continent :) ?

File: Week2/queries.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                       ON cl.CountryCode = c.Code
+                       WHERE c.Region = ?`;
+
+const sqlCities = `SELECT c.Name 
+                      FROM city c
+                      JOIN countrylanguage cl
+                      ON c.CountryCode = cl.CountryCode
+                      WHERE cl.Language = ?`;
+
+const sqlCountries = `SELECT c.name
```
Comment (mogaleaf) [inline Week2/queries.js]: good

File: Week2/queries.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+const sqlCapital = `SELECT c.Name AS capital
+                       FROM city c
+                       JOIN country ct 
+                       ON c.ID = ct.Capital
+                       WHERE ct.Name = ?`;
+
+const sqlLanguage = `SELECT distinct cl.Language
```
Comment (mogaleaf) [inline Week2/queries.js]: good

File: Week2/queries.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+const sqlCapital = `SELECT c.Name AS capital
```
Comment (mogaleaf) [inline Week2/queries.js]: good

File: Week2/queries.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+                       ON c.ID = ct.Capital
+                       WHERE ct.Name = ?`;
+
+const sqlLanguage = `SELECT distinct cl.Language 
+                       FROM countrylanguage cl 
+                       JOIN country c  
+                       ON cl.CountryCode = c.Code
+                       WHERE c.Region = ?`;
+
+const sqlCities = `SELECT c.Name
```
Comment (mogaleaf) [inline Week2/queries.js]: We want the number of cities here, not all the names :)

File: Week2/LanguageTrigger.sql [hunk 6]
```diff
@@ -0,0 +1,40 @@
+
+-- Gets alerts when a country has >= 10 languages.
+DROP TRIGGER IF EXISTS `new_world`.`countrylanguage_AFTER_INSERT`;
```
Comment (mogaleaf) [inline Week2/LanguageTrigger.sql]: your trigger is good I would just have done  it on a BEFORE than on AFTER. Better to raise error as soon as possible.

File: Week2/queries.js (hunk 1 above)
Comment (mogaleaf) [reply Week2/queries.js]: You need a distinct here if you don't want to count languages twice  (canada and usa speak english in america)

**Comment by ShyarK** [comment]
Do I need to do something here? Because I responded to the feedback!

**Pull request closed without merging**